src/modular/const_monty_form.rs

```diff
@@ -68,7 +68,7 @@ pub trait ConstMontyParams<const LIMBS: usize>:
 /// The modulus is constant, so it cannot be set at runtime.
 ///
 /// Internally, the value is stored in Montgomery form (multiplied by MOD::ONE) until it is retrieved.
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
I'm a little concerned about deriving it for potentially secret-containing types. Though the default `Hasher` is `SipHash` which is fine when properly keyed, other hashers do not necessarily prevent preimage attacks

Reply:
Do you think documenting the caveats properly is enough? A bigint library that blocks all potential security misuses risks crippling itself a fair bit and it’s not obvious where to draw the line. I’d argue that in this case it’s expected that users know about the security hazards.